front/app/containers/Admin/tools/Workshops/index.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import { Box, Text, colors } from '@citizenlab/cl2-component-library';
+import Button from 'components/UI/Button';
+import { useIntl } from 'utils/cl-intl';
+import messages from '../messages';
+import workshopsImage from './workshops.png';
+
+export const Workshops = () => {
+  const { formatMessage } = useIntl();
+
+  return (
+    <Box>
+      <Box background={colors.white} display="flex" p="20px">
+        <Box w="320px" h="240px">
+          <img
+            src={workshopsImage}
+            alt={formatMessage(messages.workshopsImage)}
+          />
+        </Box>
+
+        <Box ml="32px" display="flex" flexDirection="column">
+          <Text variant="bodyL" color="primary" my="0px">
+            {formatMessage(messages.workshopsTitle)}
+          </Text>
+          <Text color="coolGrey700">
+            {formatMessage(messages.workshopsDescription)}
+          </Text>
+          <Box display="flex" width="100%" alignItems="center">
+            <Button
+              height="45px"
+              icon="arrow-right"
+              iconColor={colors.white}
+              iconPos="right"
+              linkTo={`${window.location.origin}/workshops`}
```

Comment:
```suggestion
              linkTo='/admin/workshops'
```

Simpler? :-)

Reply:
No because we are interested in the origin. The functionality would change with your suggestion